src/sentry/static/sentry/app/components/assistant/guideAnchor.jsx

```diff
@@ -48,123 +57,134 @@ const GuideAnchor = createReactClass({
   },
 
   onGuideStateChange(data) {
-    if (
+    const active =
       data.currentGuide &&
-      data.currentStep > 0 &&
-      data.currentGuide.steps[data.currentStep - 1].target === this.props.target &&
-      // TODO(adhiraj): It would be more correct to let invisible anchors become active,
-      // and use CSS to make them invisible.
-      this.props.type !== 'invisible'
-    ) {
-      this.setState({active: true});
-    } else {
-      this.setState({active: false});
-    }
+      data.currentGuide.steps[data.currentStep].target === this.props.target;
+    this.setState({
+      active,
+      guide: data.currentGuide,
+      step: data.currentStep,
+      org: data.org,
+      messageVariables: {
+        orgSlug: data.org && data.org.slug,
+        projectSlug: data.project && data.project.slug,
+      },
+    });
+  },
+
+  interpolate(template, variables) {
+    const regex = /\${([^{]+)}/g;
+    return template.replace(regex, (match, g1) => {
+      return variables[g1.trim()];
+    });
+  },
+
+  /* Terminology:
+   - A guide can be FINISHED by clicking one of the buttons in the last step.
+   - A guide can be DISMISSED by x-ing out of it at any step except the last (where there is no x).
+   - In both cases we consider it CLOSED.
+  */
+  handleFinish(e) {
+    e.stopPropagation();
+    const {guide, org} = this.state;
+    recordFinish(guide.id, org);
+    closeGuide();
+  },
+
+  handleNextStep(e) {
+    e.stopPropagation();
+    nextStep();
+  },
+
+  handleDismiss(e) {
+    e.stopPropagation();
+    const {guide, step, org} = this.state;
+    dismissGuide(guide.id, step, org);
   },
 
   render() {
-    const {target, type} = this.props;
-    const {active} = this.state;
+    const {active, guide, step, messageVariables} = this.state;
     if (!active) {
       return this.props.children ? this.props.children : null;
     }
 
+    const body = (
+      <GuideContainer>
+        <GuideInputRow>
+          <StyledTitle>{guide.steps[step].title}</StyledTitle>
+          {step < guide.steps.length - 1 && (
+            <div
```

Comment:
You should use a `button` or `a href="#"` element for clickable things. Doing so helps screen readers and keyboard users.